modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -24,6 +24,11 @@ def fetch_aud
 
       def validate_veteran_identifiers
         return if target_veteran.participant_id.present? && target_veteran.birls_id.present?
+
+        if target_veteran.participant_id.present? && target_veteran.birls_id.blank?
+          raise ::Common::Exceptions::UnprocessableEntity.new(detail: 'No birls_id while participant_id present')
```

Comment:
The wording of this is a little weird to me. I would have an easier time interpreting this error if it was simply "birls_id is missing for the veteran" or something like that

Reply:
fair point regarding the wording, i just stole what va.gov is responding with:
https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/v0/mpi_users_controller.rb#L21

im not sure that "birls_id is missing for the veteran" gives the full picture though. a Veteran could be missing their birls_id and their participant_id and that would be fine. its specifically the situation where the Veteran has a participant_id but is missing a birls_id